src/high-level/src/KinDynComputations.cpp

```diff
@@ -84,6 +84,9 @@ struct KinDynComputations::KinDynComputationsPrivateAttributes
     // conversion is performed on set/get .
     iDynTree::FreeFloatingVel m_vel;
 
+    // Base velocity expressed in the m_frameVelRepr representation
+    iDynTree::Twist m_baseVel;
```

Comment:
Sorry, `m_baseVel` is quite confusing (a lot of attributes here are named in this way, but better not to increase the vagueness). For example, just by reading the names, almost everyone would assume that `m_baseVel` is the base part of `m_vel`, while that is not the case. I think we should have a more precise name here, for example:
```suggestion
    iDynTree::Twist m_baseVelSetViaRobotState;
```

or any alternative you prefer, thanks!

Reply:
Makes totally sense. Done with [74d9e9d](https://github.com/robotology/idyntree/pull/1164/commits/74d9e9dccb3e6523cd64830e5a57196cf487f68d)